app/Models/User.php

```diff
@@ -23,6 +23,11 @@ final class User extends Authenticatable implements MustVerifyEmail
     const MODERATOR = 2;
     const ADMIN = 3;
 
+    const LABELS = [
```

Comment:
Not sure we need this when we're rendering both as `moderator`. I think we can just use `moderator` directly in the template.

Reply:
I thought of scenario if we want to use other labels later, if not the case I'll use `moderator` directly in the template